addons/rose/app/styles/rose/components/layout/_body-content.scss

```diff
@@ -8,7 +8,7 @@
 .rose-layout-body-content {
   display: grid;
   grid-template-areas: 'body body sidebar';
-  grid-template-columns: 1fr 1fr 300px;
+  grid-template-columns: 1fr 1fr fit-content(300px);
```

Comment:
This was to originally handle the case where we might not have any target info but we _should_ always have some target info due to the user hitting the target list route first.

I kept this in here though because it still makes the grid more responsive and doesn't otherwise change anything.